node/lib/util/push.js

```diff
@@ -66,38 +67,46 @@ const EMPTY_TREE = "4b825dc642cb6eb9a060e54bf8d69288fbee4904";
  *
  * @async
  * @param {NodeGit.Repository} repo
- * @param {String}             remoteName
  * @param {NodeGit.Commit}     commit
  */
-exports.getClosePushedCommit = co.wrap(function*(repo, remoteName, commit) {
```

Comment:
could you also update the comment? The comment still talks about param `remoteName` can be used